src/mame/heathkit/tlb.h

```diff
@@ -137,7 +143,34 @@ class heath_ultra_tlb_device : public heath_tlb_device
 	void mem_map(address_map &map);
 };
 
+// Heath TLB plus Northwest Digital Systems GP-19
+class heath_gp19_tlb_device : public heath_tlb_device
+{
+public:
+	heath_gp19_tlb_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock = 0);
+
+protected:
+	virtual const tiny_rom_entry *device_rom_region() const override;
+	virtual ioport_constructor device_input_ports() const override;
+
+	virtual void device_start() override;
+	virtual void device_add_mconfig(machine_config &config) override;
+
+	void mem_map(address_map &map);
+	void io_map(address_map &map);
+
+	void latch_u5_w(uint8_t data);
+
+	virtual MC6845_UPDATE_ROW(crtc_update_row) override;
+
+	bool char_gen_a11;
+	bool graphic_mode;
+	bool col_132;
+	bool reverse_video;
```

Comment:
Why do these data members lack the `m_` prefix that the other data members in this source file all have?  It confused me when reading the body of `latch_u5_w` as they looked like locals at first glance.